labelCloud/definitions/label_format.py

```diff
@@ -0,0 +1,14 @@
+from enum import Enum
+from typing import List
+
+
+class LabelFormat(str, Enum):
```

Comment:
Need to handle segmentation model as well

Reply:
Yeah, completely forgot about that.

I propose sth. [in a comment below](https://github.com/ch-sa/labelCloud/pull/136#issuecomment-1426998973).

And here I notice that we need `ObjectDetectionLabelFormat` and `SemanticSegmentationLabelFormat`.